apps/backend/core/configuration/validation/validators/__init__.py

```
@@ -0,0 +1,106 @@
+    "RedisValidator",
+    # AI validators (CRITICAL)
+    "OpenAIValidator",
+    # Real-time validators (CRITICAL)
+    "PusherValidator",
+    # Auth validators (WARNING)
+    "ClerkValidator",
+    "SupabaseValidator",
+    # Platform validators (INFO)
+    "RobloxValidator",
+    "SentryValidator",
+]
```

Ruff (0.14.4)

`__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)